sympy/physics/units/definitions.py

```diff
@@ -368,7 +369,7 @@
 # Atomic mass
 amu = amus = atomic_mass_unit = atomic_mass_constant = Quantity("atomic_mass_constant")
 atomic_mass_constant.set_dimension(mass)
-atomic_mass_constant.set_scale_factor(1.660539040e-24*gram)
+atomic_mass_constant.set_scale_factor(1.660539060e-24*gram)
```

Comment:
The document you linked has 1.6605390666. Actually it has 1.66053906660050 if you include the digits in parentheses.

Reply:
I didn't know if there was a policy on including the uncertainty, so I used the certain digits. I could definitely update with the values in parentheses.

Reply:
I don't know what the policy is either. If none of the other constants include the uncertainty then we should leave it out.

Reply:
I'll take a look through the constants definitions and see what the standard is.

Reply:
Looking through, it was hard to find a good point of comparison since some numbers aren't included to full precision (e.g. elementary charge) and a lot of constants need to be updated.